### PR TITLE
(SIMP-1695) Fix Global Catalyst

### PR DESCRIPTION
**Pull request by jeannegreulich**
updated global catalyst
  strong typed variables
  removed NSCD options and replaced them with SSSD
  updated Gemfile

SIMP-1695 #comment complete pam module
SIMP-2182 #close
SIMP-1687 #comment pam module updated (only need simp module updated.)
SIMP-2288 #comment pupmod-simp-pam has completed
SIMP-2297 #close

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/simp/pupmod-simp-pam/pull/29" target="_blank">ReviewNinja

File: manifests/init.pp [hunk 1]
```diff
@@ -186,196 +186,126 @@
 #   Default: [ 'root' ]
 #   The users for which TTY auditing is enabled. Set to an empty Array to not audit TTY actions for any user.
 #
-# [*use_templates*]
```
Comment (jeannegreulich) [inline manifests/init.pp]: I didn't 
I changed the names because the parameter names changed.

Comment (jeannegreulich) [inline manifests/init.pp]: Sorry, I saw -use_sssd and + sssd now I see the removed comments.

File: manifests/init.pp [hunk 2]
```diff
@@ -229,153 +228,122 @@
...
+  String                  $other_content             = '',
+  String                  $fingerprint_auth_content  = '',
+  String                  $system_auth_content       = '',
+  String                  $password_auth_content     = '',
+  String                  $smartcard_auth_content    = '',
+  Boolean                 $enable                    = true,
+  Boolean                 $enable_warning            = true,
+) {
 
+  if lookup( { 'name' => 'simp_options::pam', 'default_value' => true } )
```
Comment (nick-markowski) [inline manifests/init.pp]: should be a simplib::lookup

**Comment by trevor-vaughan** [comment]
Based on recent usages of this module, it needs some breaking restructuring to be more usable to people. Will work on this tomorrow.

**Pull request merged**